app/src/main/AndroidManifest.xml

```diff
@@ -9,6 +9,7 @@
         android:roundIcon="@mipmap/ic_launcher_round"
         android:supportsRtl="true"
         android:theme="@style/AppTheme">
+        <activity android:name=".UserSettingActivity"></activity>
```

Comment:
Please lock this orientation to portrait mode
Thank you

Reply:
@CraigVitirinyuT thanks for the review - I made the changes please verify.